packages/ffe-datepicker-react/src/calendar/MonthPicker.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+
+export default function MonthPicker(props) {
+    const { previousMonthHandler, nextMonthHandler, month } = props;
+
+    function left(evt) {
+        previousMonthHandler(evt);
+    }
+
+    function right(evt) {
```

Comment:
`left` and `right` seems like very unnecessary wrappers.

Reply:
I had an intention of activating an outline around the MonthPicker, ie. setting it to focused when clicked. I figured this may be implemented in these function wrappers. They are unnecessary as they are now.